web/src/app/browser/src/keymanEngine.ts

```diff
@@ -175,6 +175,19 @@ export default class KeymanEngine extends KeymanEngineBase<BrowserConfiguration,
 
     await super.init(totalOptions);
 
+    // Used by keymanweb.com; if no keyboard-cookie exists, we need this to trigger
```

Comment:
Really only used by keymanweb.com? Not by almost every single site that uses keymanweb? After all, no sites will have cookies on initial use.

Reply:
Most sites don't directly call `keyman.register`.

keymanweb.com's `cdn/prog/keyboards.php` resolves to this due to caching:

```javascript
/* tier alpha, version 17.0, cached 2023-10-18T02:25:03+00:00 */
function addKeyboards() {
    keyman.register({
        "options": {
// ...
```

which is then called here:

https://github.com/keymanapp/keymanweb.com/blob/0dba6472d3300ef15a4820ee53953d0051333aa0/inc/head.php#L207-L215

```javascript
  keyman.init({
    attachType:'auto',
    setActiveOnRegister: setActiveOnRegister
  }).then(function() {
    if(typeof afterInit == 'function') {
      afterInit();
    }
    if(typeof addKeyboards == 'function') {
      addKeyboards();
```

The site completely bypasses KeymanWeb's `addKeyboards` API, using an unofficial, undocumented optimization.  Therefore...

> not by almost every single site that uses keymanweb